app/gui/src/main/scala/org/bitcoins/gui/WalletGUI.scala

```diff
@@ -88,7 +109,11 @@ object WalletGUI extends JFXApp {
     scene = new Scene(rootView)
   }
 
-  private val taskRunner = new TaskRunner(resultArea, glassPane, statusLabel)
+  if (GlobalData.defaultDarkTheme) {
+    stage.scene.value.getStylesheets.add("/themes/dark-theme.css")
+  }
```

Comment:
I think it would be nicer to define the `scene` as a private val above the stage and do
```
  private val scene = new Scene {
    root = rootView
    stylesheets = Seq("/themes/dark-theme.css")
  }
```